packages/core/src/services/db/backends/ephem-db.provider.ts

```diff
@@ -103,6 +108,19 @@ export class EphemDBProvider implements BaseDBProvider {
     const current = (await this.get(key, [])) as T[]
     value = Array.isArray(value) ? value : [value]
     current.concat(value)
-    await this.set(key, current)
+    await this.put(key, current)
+  }
+
+  /**
+   * Accepts a string or Buffer and returns a string representation of
+   * the input.
+   * @param value
+   * @returns the value as a string
+   */
+  private convertToString(value: Buffer | string) {
+    if (typeof value === 'string')
```

Comment:
`prettier` will probably complain about missing brackets here.

Reply:
You're right! There's a bunch of linting errors -- need to update my editor's linter to follow prettier instead of whatever it's using now

Reply:
You can also run `yarn fix` in the root to fix issues automatically :+1: